samples/bot-all-cards/csharp/BotAllCards/Dialogs/AllCards.cs

```diff
@@ -0,0 +1,157 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.Bot.Schema;
+using Microsoft.Bot.Schema.Teams;
+using Newtonsoft.Json;
+
+namespace BotAllCards.Dialogs
+{
+    public static class AllCards
+    {
+        /// <summary>
+        /// Adaptive Card
+        /// </summary>
+        /// <returns></returns>
```

Comment:
Please add proper summary for this method.

Reply:
Fixed